shell/platform/common/client_wrapper/include/flutter/method_channel.h

```diff
@@ -91,9 +91,8 @@ class MethodChannel {
   // Registers a handler that should be called any time a method call is
   // received on this channel. A null handler will remove any previous handler.
   //
-  // Note that the MethodChannel does not own the handler, and will not
-  // unregister it on destruction, so the caller is responsible for
```

Comment:
Rather than removing the statement about unregistering, maybe we could say "so the caller is responsible for unregistering explicitly if the handler stops being valid before the engine is destroyed."

While it's true that in the lifetime model used by many plugins that never happens, it's absolutely possible to use channels in ways where it can, leaving the handler with a dangling pointer.

Reply:
Thanks, added!